src/main/java/Helloworld/HelloWorld.java

```diff
@@ -1,10 +1,14 @@
 class HelloWorld
 {
-	private String  msg="hello world";
-	
-	public String sayHello()
+	String val="hello world ";
```

Comment:
This should be a constant (static final). Remember to use all upper cases for variable names that are static final.